README.md

```diff
@@ -58,8 +60,26 @@ Here are the parameters that the tool accepts:
 
 Example usage to convert HL7 v2 messages to FHIR resources in a folder:
 ```
->.\Microsoft.Health.Fhir.Liquid.Converter.Tool.exe -d myTemplateDirectory -e ADT_A01 -i myInputDataFolder -o myOutputDataFolder
+>.\Microsoft.Health.Fhir.Liquid.Converter.Tool.exe convert -d myTemplateDirectory -e ADT_A01 -i myInputDataFolder -o myOutputDataFolder
+```
+
+**Manage Templates**
```

Comment:
Let's move the 'manage template' under the '$convert-data operation in the Azure API for FHIR' section with appropriate context. This information is relevant there.

Reply:
This part is the usage of CLI for template management. Seems in 'command line tool' section is more property?

Reply:
I see; we can keep it here.